src/zenml/new/pipelines/pipeline.py

```diff
@@ -761,6 +768,13 @@ def _run(
                         "`zenml up`."
                     )
 
+                if self.configuration.model:
+                    logs_of_model = self.configuration.model._prepare_model_version_inside_run(
```

Comment:
Maybe the method name is wrong, but at this point we aren't actually inside the run yet right?

Reply:
Addressed